react/ProductList.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { FC } from 'react'
+import { Product } from 'vtex.catalog-graphql'
```

Comment:
I mean, we don't use the app `vtex.catalog-graphql` we just use `vtex.search-graphql` or `vtex.store-graphql`

Shouldn't we use one of those?

Reply:
Since you use different apps, and this component uses only two product properties I thought it would be best to create a specific product type with only those properties.